docs/examples/networking/nat/user_data.tpl

```diff
@@ -0,0 +1,18 @@
+#
```

Comment:
What's the simplest way to verify that this is working correctly?

Reply:
Addressed this in the README.